glean_parser/parser.py

```diff
@@ -243,6 +245,10 @@ def _instantiate_metrics(
 
             if metric_obj is not None:
                 metric_obj.no_lint = list(set(metric_obj.no_lint + global_no_lint))
+                if len(global_tags) > 0:
```

Comment:
```suggestion
                if len(global_tags):
```

not-zero is truthy (so are empty arrays, but `if len(...)` seems to be used throughout the codebase already)